test/e2e/specs/default-config/recordedit/multi-form-input.spec.js

```diff
@@ -0,0 +1,732 @@
+        chaisePage.clickButton(toggleBtn)
+          .then(() => recordEditPage.getInputSwitchContainer('markdown_col', 1))
+          .then((inputSwitch) => recordEditPage.getParentElement(inputSwitch))
+          .then((parentElement) => {
+            expect(parentElement.getAttribute("class")).toContain('entity-active', 'Form is not selected');
+            done();
+          })
+          .catch(chaisePage.catchTestError(done));
+      });
+
+      it('the newly cloned form should not be selected.', (done) => {
+        const inputSwitch = recordEditPage.getInputSwitchContainer('markdown_col', 3)
+        const parentElement = recordEditPage.getParentElement(inputSwitch);
+        chaisePage.clickButton(cloneFormSubmitButton)
+          .then(() => {
+            expect(parentElement.getAttribute("class")).not.toContain('entity-active', 'Form is selected');
+            done();
+          })
+          .catch(chaisePage.catchTestError(done));
+      });
+
+
+      it('the form should work as a toggle when selected', (done) => {
+        const inputSwitch = recordEditPage.getInputSwitchContainer('markdown_col', 3)
+        const parentElement = recordEditPage.getParentElement(inputSwitch);
+        chaisePage.clickButton(parentElement).then(() => {
+          expect(parentElement.getAttribute("class")).toContain('entity-active', 'Form is not selected');
+        }).then(() => {
+          return chaisePage.clickButton(parentElement).then(() => {
+            expect(parentElement.getAttribute("class")).not.toContain('entity-active', 'Form is selected');
+            done();
+          }).catch(chaisePage.catchTestError(done));
+        })
+      });
+      it('previous selection should remain after closing and opening again', (done) => {
+        const inputSwitch = recordEditPage.getInputSwitchContainer('markdown_col', 3)
+        const parentElement = recordEditPage.getParentElement(inputSwitch);
+        chaisePage.clickButton(parentElement).then(() => {
+          expect(parentElement.getAttribute("class")).toContain('entity-active', 'Form is not selected');
+        }).then(() => {
+          let toggleBtn = recordEditPage.getColumnMultiFormButton('markdown_col');
+          chaisePage.clickButton(toggleBtn)
+            .then(() => chaisePage.clickButton(toggleBtn))
+            .then(() => {
+              expect(parentElement.getAttribute("class")).toContain('entity-active', 'Form is not selected');
+              done();
+            })
+        }).catch(chaisePage.catchTestError(done));
+      })
+
+      it('the form should not be clickable', (done) => {
+        let toggleBtn = recordEditPage.getColumnMultiFormButton('markdown_col');
+        const inputSwitch = recordEditPage.getInputSwitchContainer('markdown_col', 1)
+        const parentElement = recordEditPage.getParentElement(inputSwitch);
+        chaisePage.clickButton(toggleBtn).then(() => {
+          return chaisePage.clickButton(parentElement).then(() => {
```

Comment:
Chain this promise with the previous ones instead of creating a new promise chain.